cmd/autoheal/job_runner.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	alertmanager "github.com/openshift/autoheal/pkg/alertmanager"
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha2"
+)
+
+type JobRunnerType int
+
+const (
+	AWX   JobRunnerType = 0
```

Comment:
I'd prefer to avoid this kind of constants, as the fact that there are different types of job runners is already indicated by the existence of the corresponding Go types. If the constants are needed then please consider using `reflect.TypeOf` or `reflect.TypeOf(...).Name()` instead.

Also, for constants try to use camel case, and add a prefix that corresponds to the type name:

```go
const (
   JobRunnerTypeAWX JobRunnerType = iota
   JobRunnerTypeBatch
)
```